packages/account-v2/src/containers/POAForm/CommonMistakeExample/CommonMistakeExampleConfig.tsx

```diff
@@ -1,9 +1,12 @@
-import IcBlurryDocument from '../../../assets/common-mistake-examples/ic-blurry-document.svg';
-import IcCroppedDocument from '../../../assets/common-mistake-examples/ic-cropped-document.svg';
-import IcDocumentAddressMismatch from '../../../assets/common-mistake-examples/ic-document-address-mismatch.svg';
-import IcDocumentNameMismatch from '../../../assets/common-mistake-examples/ic-document-name-mismatch.svg';
-import IcEnvelope from '../../../assets/common-mistake-examples/ic-envelop.svg';
-import IcOldIssuedDocument from '../../../assets/common-mistake-examples/ic-old-issued-document.svg';
+import React from 'react';
+import {
+    DerivLightIcBlurryDocumentIcon,
+    DerivLightIcCroppedDocumentIcon,
+    DerivLightIcDocumentAddressMismatchIcon,
+    DerivLightIcDocumentNameMismatchIcon,
+    DerivLightIcEnvelopIcon,
```

Comment:
You might raise it up to quill icons team :) 👀 
```suggestion
    DerivLightIcEnvelopeIcon,
```